cmd/internal/ec2/add.go

```diff
@@ -162,6 +173,20 @@ func checkSystemdExists() error {
 	return nil
 }
 
+func getAgentInstallPath() (string, error) {
+	message := "Checking agent install path "
+	printer.Infof(message + "\n")
+	reportStep(message)
+
+	for _, path := range agentInstallPaths {
+		if _, err := exec.LookPath(path); err == nil {
```

Comment:
I wonder whether we should be using LookPath directly, for just "postman-insights-agent", and using whichever is first in the path?  Failing that, maybe use argv[0]?

I don't think there's anything necessarily wrong with building in the set of paths that we know the install script uses today, but it does mean that the two have to be kept in sync.  So that's why I'm wondering whether we could be more flexible in terms of "wherever in the path postman-insights-agent" is installed or "wherever you successfully ran the agent" -- the latter seems very reasonable to me if we can make it work?

Reply:
I have added a check for the default agent path also.